.github/settings.yml

```diff
@@ -26,3 +26,62 @@ collaborators:
     permission: admin
   - username: kapunahelewong
     permission: admin
+
+# Labels: define labels for Issues and Pull Requests
+labels:
+  - name: cleanup
+    color: 1ebcd8
+  - name: e1-hours
+    description: "Effort: < 8 hrs"
+    color: d9b38c
+  - name: e2-days
+    description: "Effort: < 5 days"
+    color: cc9966
+  - name: e3-weeks
+    description: "Effort: < 4 weeks"
+    color: bf8040
+  - name: e4-months
+    description: "Effort: 1+ months"
+    color: 996633
+  - name: infrastructure
+    color: 696969
+  - name: p0-critical
+    color: B60205
+  - name: p1-high
+    color: D93F0B
+  - name: p2-medium
+    color: FBCA04
+  - name: p3-low
+    color: FEF2C0
+  - name: e0-minutes
+    description: "Effort: < 60 min"
+    color: e6ccb3
+  # Default new repo labels
+  - name: bug
+    description: "Something isn't working"
+    color: d73a4a
+  - name: documentation
```

Comment:
Hmm, let's drop this (like we did for etcd.io) -- since most PRs over a website repo will be for documentation (i.e., let's take that as the default when there's no extra label).

Reply:
I was thinking about that -- but as it is one of the default labels that GitHub creates on repo creation, I wonder if there is some value in keeping it (will template users expect it, even if we don't use it).